common/djangoapps/student/admin.py

```diff
@@ -197,8 +197,17 @@ class Meta(object):
         model = UserAttribute
 
 
+@admin.register(CourseEnrollmentAllowed)
+class CourseEnrollmentAllowedAdmin(admin.ModelAdmin):
+    """ Admin interface for the CourseEnrollmentAllowed model. """
+    list_display = ('email', 'course_id', 'auto_enroll')
+    search_fields = ('email', 'course_id')
+
```

Comment:
OK, changed that.  (Some people prefer the trailing comma since it avoids a pitfall when reducing the length of the tuple to one.  It's not done consistently in edx-platform, but seems to be done in this particular file.)